integration/entity.go

```diff
@@ -59,14 +67,20 @@ func newEntity(name, namespace string, storer persist.Storer, addHostnameToMetad
 		lock:        &sync.Mutex{},
 	}
 
-	// Entity data is optional. When not specified, data from the integration is reported for the agent's own entity.
-	if name != "" && namespace != "" {
-		d.Metadata = &EntityMetadata{
-			Name:      name,
-			Namespace: namespace,
+	idAttrs := make([]map[string]string, len(attributes))
+	for i := 0; i < len(attributes); i++ {
```

Comment:
why not range?

Reply:
As I was assigning using `index` instead of `append` for readability reasons I chose access by index in both sides of the operation.

But I'll change it using `range`